app/views/dashboard/_discovery_widget.html.erb

```diff
@@ -1,10 +1,12 @@
-<h4 class="ca"><%= _('Discovered Host Pool') %> - <%= @current_permission %></h4>
-<% all_hosts = Host::Discovered.includes(:model, :interfaces).includes(:discovery_attribute_set).scoped %>
-<% hosts = all_hosts.limit(5) %>
+<% all_hosts = Host::Discovered.includes(:model).includes(:discovery_attribute_set).scoped %>
+<% hosts = all_hosts.order('created_at DESC').limit(6) %>
```

Comment:
Can you implement the same ordering for the main discovered hosts index page as well? That would be useful. To be honest, I think we are currently unsorted.